### PR TITLE
Feature/grv 1419 service allowed voter

### PR DESCRIPTION
**Pull request by lapistano**

### CODE REVIEW

**Comment by hairmare** [comment]
There seems to be some re-adds due to merge conflicts from #40.

Could you check those.

File: src/Graviton/SecurityBundle/README.md [hunk 1]
```diff
@@ -59,7 +59,52 @@ The service referenced in the parameter must implement the »\Graviton\RestBundl
...
+  
+// $request received from ParameterConverter of the action
+if (false === $authorizationChecker->isGranted('VIEW', $account)) {
+    throw new AccessDeniedException('You are not allowed to be here.');
+}  
+```
+
+- ServiceAllowedVoter
+Acting on the Request object (Symfony\Component\HttpFoundation\Request) this voter determines depending
+on a configured whitelist (graviton.security.services.whitelist), if a service may be called or not.
```
Comment (hairmare) [inline src/Graviton/SecurityBundle/README.md]: Will we be adding to this whitelist on a wrapper level through ``parameters.yml``? We should start making that possible in wrappers.

**Comment by lapistano** [comment]
mmh .. the only conflict I got was in the README.md ..

**Comment by hairmare** [comment]
You are re-adding ``app/config/parameters.yml``. I haven't checked for more re-adds, sadly those never end up as a conflict :(

not sure if git bug...

**Pull request merged**